src/app/dev/DevToys.Tools/DevToys.Tools.csproj

```diff
@@ -398,6 +407,9 @@
   </ItemGroup>
 
   <ItemGroup>
+    <PackageReference Include="Microsoft.IdentityModel.JsonWebTokens" />
+    <PackageReference Include="Newtonsoft.Json" />
```

Comment:
There's already **Newtonsoft.Json** on line 420.

Reply:
Oh sorry don't know why it's added twice.